config/default.yaml

```diff
@@ -132,7 +132,12 @@ scm:
         # The username and email used for checkout with bitbucket
         #username: sd-buildbot
         #email: dev-null@screwdriver.cd
-
+    gitlab:
+        oauthClientId: YOUR-GITLAB-OAUTH-CLIENT-ID
+        oauthClientSecret: YOUR-GITLAB-OAUTH-CLIENT-SECRET
+        # The username and email used for checkout with gitlab
+        #username: sd-buildbot
+        #email: dev-null@screwdriver.cd
```

Comment:
may want to add in in comments that the plugin also supports going to on-premise gitlab hosts as well.

```
## if you have on-premise gitlab, you can specify that here
gitlabHost: myawesomegitlab.com
gitlabProtocol: https
```

gitlab.com is default